pkg/util/flag/flags.go

```diff
@@ -52,7 +52,7 @@ func (m *ConfigValue) Set(s string) error {
 }
 
 func (m *ConfigValue) String() string {
-	return fmt.Sprintf("%v", bool(*m == ConfigTrue))
```

Comment:
This file is from K8s repo, let's keep it as above :) Ref: https://github.com/kubernetes/kubernetes/blob/master/pkg/util/flag/flags.go

Reply:
done